renderdoc/driver/vulkan/wrappers/vk_resource_funcs.cpp

```diff
@@ -2477,7 +2512,18 @@ VkResult WrappedVulkan::vkCreateImage(VkDevice device, const VkImageCreateInfo *
            next->sType == VK_STRUCTURE_TYPE_EXTERNAL_FORMAT_ANDROID)
         {
           isExternal = true;
-          break;
+
+          // we can't call vkGetImageMemoryRequirements on AHB-backed images until they are bound
+          if(next->sType == VK_STRUCTURE_TYPE_EXTERNAL_MEMORY_IMAGE_CREATE_INFO)
+          {
+            VkExternalMemoryImageCreateInfo *extCreateInfo =
+                (VkExternalMemoryImageCreateInfo *)FindNextStruct(
+                    &createInfo_adjusted, VK_STRUCTURE_TYPE_EXTERNAL_MEMORY_IMAGE_CREATE_INFO);
```

Comment:
Could we cast `next` directly here rather than using `FindNextStruct`, since we've effectively done our own search here?